opencog/atomspace/AtomSpace.cc

```diff
@@ -495,6 +495,22 @@ Handle AtomSpace::set_truthvalue(const Handle& h, const TruthValuePtr& tvp)
     return Handle::UNDEFINED;
 }
 
+bool AtomSpace::is_in_atomspace(Handle const & h){
```

Comment:
This has a trivial one-line implementation: `return nullptr != get_atom(h);` -- therefore, this new code is not needed.  You should just copy this one-liner to the python code.